src/actions/posts.js

```diff
@@ -519,75 +537,18 @@ export function getPostThread(postId, skipAddToChannel = true) {
             return {error};
         }
 
-        const post = posts.posts[postId];
-
         dispatch(batchActions([
-            {
-                type: PostTypes.RECEIVED_POSTS,
-                data: posts,
-                channelId: post.channel_id,
-                skipAddToChannel,
-            },
+            receivedPosts(posts),
+            receivedPostsInThread(posts, rootId),
             {
                 type: PostTypes.GET_POST_THREAD_SUCCESS,
             },
-        ], 'BATCH_GET_POST_THREAD'), getState);
+        ]), getState);
```

Comment:
`getState` is not needed here

Reply:
Once we're all unblocked with this redux stuff, I'm going to look at doing a mass refactor to get rid of this once and for all